hloc/extractors/disk.py

```diff
@@ -0,0 +1,109 @@
+import sys
+from pathlib import Path
+from functools import partial
+import torch
+import torch.nn.functional as F
+
+from ..utils.base_model import BaseModel
+
+disk_path = Path(__file__).parent / "../../third_party/disk"
+sys.path.append(str(disk_path))
+from disk import DISK as _DISK
+
+class DISK(BaseModel):
+    default_conf = {
+        'model_name': 'depth-save.pth', # Name of the model's .pth save file
+        # we rescale image only in the preprocessing step.
+        # we only pad the image size to be the multiple of 16 in this model
+        # 'height': None, # rescaled height (px). If unspecified, image is not resized in height dimension
+        # 'width': None, # rescaled width (px). If unspecified, image is not resized in width dimension
+        'window':  5, # NMS window size
+        'n': None, # Maximum number of features to extract. If unspecified, the number is not limited
+        'desc-dim': 128, # descriptor dimension. Needs to match the checkpoint value
+        'mode': 'nms', # Whether to extract features using the non-maxima suppresion mode or through training-time grid sampling technique
```

Comment:
It would be great to clean up the comments and make the key names consistent with the other extractors (see my comments on the R2D2 PR).